frontend/src/util/validators.js

```diff
@@ -292,6 +333,7 @@ export const VALIDATE_FIRST_NAME_MESSAGE = 'First Name is invalid'
 export const VALIDATE_SECOND_NAME_MESSAGE = 'Second Name is invalid'
 export const VALIDATE_CITY_MESSAGE = 'City is invalid'
 export const VALIDATE_DOB_MESSAGE = 'Date of Birth must not be in the future'
+export const VALIDATE_MINIMUM_DATE_MESSAGE = 'Date must be later than 19000101'
```

Comment:
I don't think this is quite correct as 19000101 is a valid date. "Date must be on or after 19000101".